extension/js/common/composer.ts

```diff
@@ -127,6 +127,7 @@ export class Composer {
       await this.initComposeBox();
```

Comment:
I'm not sure this part will do what you expect.

I think, if `await this.initComposeBox();` fails, you may have an exception that gets silently swallowed. Either way, as is, with async resolver inside a Promise, it's not 100% clear what will happen from the code.

My recommendation is to refactor the whole thing as simply an async self-invoked function:

```js
 this.initialized = (async () => {
      await this.initComposeBox();
      await this.initActions();
      await this.checkEmailAliases();
})();
```
This returns a promise, which you can then await elsewhere.

Reply:
Totally agree